net-misc/seafile/seafile-8.0.6.ebuild

```
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Maybe we could bump to EAPI-8 next time :)

```suggestion
EAPI=8
```